docs/requirements.txt

```diff
@@ -6,48 +6,58 @@
 #
 alabaster==0.7.16
     # via sphinx
-babel==2.14.0
+anyio==4.4.0
+    # via
+    #   starlette
+    #   watchfiles
+babel==2.15.0
     # via sphinx
-certifi==2023.11.17
+certifi==2024.6.2
     # via requests
 charset-normalizer==3.3.2
     # via requests
+click==8.1.7
+    # via uvicorn
 colorama==0.4.6
     # via sphinx-autobuild
 docutils==0.19
     # via
     #   -r requirements.in
     #   sphinx
+exceptiongroup==1.2.1
+    # via anyio
+h11==0.14.0
+    # via uvicorn
 idna==3.7
-    # via requests
+    # via
+    #   anyio
+    #   requests
 imagesize==1.4.1
     # via sphinx
-jinja2==3.1.4
+jinja2==2.11.3
```

Comment:
Why are we going down in the version here?

Reply:
Causes a version conflict, see https://github.com/superdesk/superdesk-core/pull/2607. This should be fixed when we upgrade Flask etc